docs/livepeer-in-a-box.md

```diff
@@ -1,10 +1,95 @@
 # Livepeer in a Box
 
-Livepeer in a Box is our development environment for the full Livepeer stack,
-including Livepeer Studio and Livepeer Catalyst. We currently support Linux and
-macOS hosts.
+Livepeer in a Box allows you to run the full Livepeer Studio stack as a single
+Docker image. This stack includes Catalyst, MistServer, the Studio API server,
+and a packaged Studio frontend.
+
+What the box does:
+
+-   Boots up a full-stack Livepeer experience on your laptop with a single
+    command
+-   Facilitates easy development of various components of the Livepeer stack
+-   Allows for development of applications against the Livepeer Studio API
+    locally that can then transfer to the hosted version at
+    [livepeer.studio](https://livepeer.studio) when you're ready to go to
+    production
+-   Bundles a fully-local offchain go-livepeer broadcaster and orchestrator, so
+    that you may test transcoding with no external dependencies
+
+What the box doesn't do (yet):
+
+-   Allow for easy deployment to a server. There are presently many hardcoded
+    references to `localhost`, and these things will break in any other
+    environment.
+-   No multi-user streaming (Livekit) integration
+-   No usage or billing data
+-   No GPU transcoding support. We recommend using very low-bitrate test files,
+    especially if running the box using Docker for Mac or Docker for Windows.
+    The built-in profiles for livestream transcoding use a single 240p
+    low-quality rendition.
+
+## Running the Box
+
+First, select a directory for persisting your database and video content; in
+this example we will be using `$HOME/livepeer-in-a-box`.
 
-## Dependencies
+```shell
+BOX_DIR="$HOME/livepeer-in-a-box"
+mkdir -p $BOX_DIR
+docker run \
```

Comment:
Should we encourage to use one of `make` commands?

Reply:
No, 'cause these are directions for someone that might be running it without developing it